pl-PL.json by JamesBłąd

JamesBlad opened the pull request:
I did a Polish translation. Not everything could be translated. I hope it will increase the number of users in Poland :D Regards JamesBłąd

On BeamMP JamesBond007PL

JamesBlad committed:
pl-PL.json by JamesBłąd

I did a Polish translation. Not everything could be translated. I hope it will increase the number of users in Poland :D
Regards JamesBłąd

On BeamMP JamesBond007PL

Starystars67 commented:
Hey, thank you for raising this. I notice that there are a number of lines which are not translated yet, Are you able to translate them too please?

JamesBlad commented:
Oh sorry I upload bad copy. Shortly I rapair my mistake.

JamesBlad committed:
Update pl-PL.json

I update the Translation. Now is correct. Regards JamesBłąd, On BeamMP JamesBond007PL

Starystars67 reviewed (approved):
Thank you for contributing towards the BeamMP mod :)
